test/java/seedu/addressbook/commands/AddCommandTest.java

```diff
@@ -49,7 +49,7 @@ public void addCommand_invalidPhone_throwsException() {
     @Test
     public void addCommand_invalidEmail_throwsException() {
         final String[] invalidEmails = { "", " ", "def.com", "@", "@def", "@def.com", "abc@",
-                                         "@invalid@email", "invalid@email!", "!invalid@email" };
+                                         "@invalid@email", "invalid@email!", "!invalid@email", "invalid@email", "email@invalid_domain_name" };
```

Comment:
This line is too long -- the style guide specified 142 characters. You should break this line into multiple lines.